packages/eslint-plugin-template/src/rules/headings-consecutive-order.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  TmplAstElement,
+  TmplAstTemplate,
+} from '@angular-eslint/bundled-angular-compiler';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { getTemplateParserServices } from '@angular-eslint/utils';
+import { TmplAstNode } from '@angular/compiler';
+
+export const RULE_NAME = 'headings-consecutive-order';
+export type MessageId = 'headingsOrder';
+
+export default createESLintRule<[], MessageId>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      // link: https://www.w3.org/WAI/tutorials/page-structure/headings/
+      description:
+        'Ensures that headings have consecutive order from largest to lowest h1 -> h6',
+      recommended: false,
+    },
+    schema: [],
+    messages: {
+      headingsOrder: 'Headings number sequence must not be skipped',
+    },
+  },
+  defaultOptions: [],
+  create(context) {
+    const parserServices = getTemplateParserServices(context);
+    let headingLastIndex: number;
+    let ngTemplateChildrenTagsToSkip: string[] = [];
+
+    return {
+      [`Element$1[name=/^(h1|h2|h3|h4|h5|h6)$/], Template[tagName="ng-template"]`](
+        element: TmplAstElement | TmplAstTemplate,
+      ) {
+        if (element instanceof TmplAstTemplate) {
+          // "ng-template"
+          if (element.children) {
+            const hTags = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
+
+            element.children.forEach((child: TmplAstNode) => {
+              if (
+                child instanceof TmplAstElement &&
+                hTags.includes(child.name)
+              ) {
+                ngTemplateChildrenTagsToSkip.push(child.name);
+              }
+            });
+          }
+        }
+
+        if (element instanceof TmplAstElement) {
+          // headings
+          if (
+            ngTemplateChildrenTagsToSkip.length > 0 &&
+            element.name === ngTemplateChildrenTagsToSkip[0]
+          ) {
+            ngTemplateChildrenTagsToSkip.shift();
+            return;
+          }
+
+          const headingCurrentIndex = +element.name.slice(1);
+
+          if (
+            headingLastIndex &&
+            headingCurrentIndex - headingLastIndex !== 1 &&
+            headingCurrentIndex !== 1 &&
+            headingCurrentIndex > headingLastIndex
```

Comment:
Is the `headingCurrentIndex !== 1` necessary? Seems `headingCurrentIndex` could not be both `1` && `> headingLastIndex`
```suggestion
            headingCurrentIndex > headingLastIndex
```

Reply:
Thanks for the reviews, Sandi :)